src/Typist.jsx

```diff
@@ -39,6 +41,7 @@ export default class Typist extends Component {
   state = {
     text: [],
     isDone: false,
+    flag: true,
```

Comment:
what is this `flag` state used for? is it actually necessary? and if so, can we give it a better name?